components/icon-picker/icon-picker-toolbar-button.js

```diff
@@ -21,18 +21,18 @@ const StyledIconPickerDropdown = styled(IconPicker)`
  * @returns {*}
  */
 export const IconPickerToolbarButton = (props) => {
-	const {
-		value: { name, iconSet },
-		buttonLabel,
-	} = props;
+	const { value, buttonLabel } = props;
 
-	const buttonIcon = name && iconSet ? <Icon name={name} iconSet={iconSet} /> : null;
+	const buttonIcon =
+		value?.name && value?.iconSet ? <Icon name={value?.name} iconSet={value?.iconSet} /> : null;
 
 	return (
 		<Dropdown
 			className="component-icon-picker-toolbar-button"
 			contentClassName="component-icon-picker-toolbar-button__content"
-			position="bottom right"
+			popoverProps={{
+				placement: 'bottom-start',
+			}}
```

Comment:
`position` is deprecated since 6.2, but when was support for `popoverProps` added? I don't see this noted in the documentation for `<Dropdown>`: https://developer.wordpress.org/block-editor/reference-guides/components/dropdown/

We should consider doing the following:

1. Test for backwards compatibility with 6.1 and 6.0, for the projects that haven't upgraded to 6.2 yet. (I know of at least one `:/`)
2. File a PR adding a note to https://github.com/WordPress/gutenberg/edit/trunk/packages/components/src/dropdown/README.md about when `popoverProps` was added, and that it replaces the deprecated `position`. (Core's Gutenberg docs aren't great about capturing changes over time, in comparison to Core's PHP docs which have things like [`@since 5.5.0` on the third parameter of `get_template_part()`](https://developer.wordpress.org/reference/functions/get_template_part/))

Reply:
@benlk I looked into this and the `popoverProps` prop has been there from the very beginning https://github.com/WordPress/gutenberg/pull/14867 :) So we should be all good here

Reply:
Excellent news!